emulators/xhyve/Portfile

```diff
@@ -6,6 +6,7 @@ PortGroup           xcode 1.0
 
 github.setup        machyve xhyve eab8ad838868205b872b93129e2ee91ca5328ea9
 version             20210922
+revision            1
```

Comment:
if you are fixing a build issue there is no need to increase the revision as nothing will change for user that managed to build it already. There is a list of examples on when the revision should or shouldn't be increased in the MacPorts guide.

Reply:
@reneeotten I understand. Anyway, I’d like to rebuild it for all platforms to confirm that my fix doesn’t broke another platforms.

Reply:
that's ideally something you would do before pushing the change 😉. Or you wait for upstream to see if they agree with the fix. Third option would be to only apply the patch to affected systems and leave others as-is. I'm not sure just increasing the revision is the way to go here..

Reply:
@reneeotten @reneeotten I have tested it on 11 and 10.15 which I have and it works. Anyway, I haven't got 10.10 and 10.11 to test it.

Regarding upstream... I not sure then it was merged, because before yesterday's merge of a few PR changes was at 5th March. Also, upstream hasn't got any CI and merge may just happen without any testing.

To be blind: I'm using MacPort's as a way to proof that everything is OK. Anyway, it is OK for 10.15 and 11 on my side and CI's => let me remove increased revision to make everything clear and by the book.

Reply:
This change looks pretty benign, FWIW. The entirety of the patch removes a warning on platforms that don't understand it.

Reply:
@pmetzger I just afraid that some Compiler my not understand `-Wunknown-pragmas` because I have no idea when it was added to (Apple) clang.

Reply:
You can check by looking at the LLVM version control repository.

Reply:
@pmetzger I've tried that but it quite unclear: https://github.com/search?q=org%3Allvm+unknown-pragmas&type=code

For example inside their build system the uses it since https://github.com/llvm/llvm-project/commit/f1ddf431b5dd47a6c03ed1420af69facace8cb1d

Thus, I feel it is very safe changes and it just my paranoia.

Reply:
At worst it wasn't working on Yosemite before and it will continue not working on it, right?

Reply:
@pmetzger yes, Yosemite hasn't got `Hypervisor.framework` which is mandatory to run it.